projects/i18n/languages/vietnamese/core.ts

```diff
@@ -0,0 +1,25 @@
+import {LanguageCore} from '@taiga-ui/i18n/interfaces';
+import {TUI_VIETNAMESE_LANGUAGE_COUNTRIES} from './countries';
+
+export const TUI_VIETNAMESE_LANGUAGE_CORE: LanguageCore = {
+    months: [
+        'Tháng 1',
+        'Tháng 2',
```

Comment:
Sorry if this is a stupid question.
I know nothing in vietnamese language. That is why i am surprised by this namings of months.

The names of months are just "month" in vietnamese + index of the month, are not they ? Does months have more contrast names like "January", "February", and etc ?

Reply:
Hi,

Yes, we use 'month + index' for every month in a year, but there's another name for January and December that are "tháng giêng" and "tháng chạp".

Reply:
got it, thanks 👌